src/components/dialogs/network-modifications/vsc/converter-station/converter-station-utils.tsx

```diff
@@ -193,7 +193,7 @@ export function getVscConverterStationEmptyFormData(
 ) {
     return {
         [id]: {
-            [CONVERTER_STATION_ID]: null,
+            [CONVERTER_STATION_ID]: isModification ? '' : null,
```

Comment:
Why not always null

Reply:
this is done to avoid sending a null to the backend, which can happen when the vsc is created in the same node as the modification.